### DISCUSSION TITLE
Question on params

### DISCUSSION THREAD

**Discussion by Zarrsito** [digitalninja-ro/klipper-neopixel#26] [Q&A]
Hello,

i have 3 rings with each 16leds, im wondering if i have to adjust those paras in the macros ? 
param_led_num:  0
param_led_total: 1
?

Some sort of an Manual would be nice if something is adjustable , or has to be adjusted
also is it possible to change the colors used in templates somehow ?

Have a nice Day =)

**Comment by digitalninja-ro** [comment]
Just use the macro. You don't need to adjust any parameters.
